### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,15 +14,15 @@
 </p>
 
 # Facebook Ads Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_facebook_ads/))
-# 📣 What does this dbt package do?
+## What does this dbt package do?
 - Produces modeled tables that leverage Facebook Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/facebook-ads) in the format described by [this ERD](https://fivetran.com/docs/applications/facebook-ads#schemainformation) and builds off the output of our [Facebook Ads source package](https://github.com/fivetran/dbt_facebook_ads_source).
 - Enables you to better understand the performance of your ads across varying grains:
-  - Providing an account, campaign, ad group, keyword, ad, and utm level reports.
+- Providing an account, campaign, ad group, keyword, ad, and utm level reports.
```
Comment (fivetran-joemarkiewicz) [inline README.md:20]:
```suggestion
  - Providing an account, campaign, ad group, keyword, ad, and utm level reports.
```

File: README.md [hunk 2]
```diff
@@ -14,15 +14,15 @@
...
 - Materializes output models designed to work simultaneously with our [multi-platform Ad Reporting package](https://github.com/fivetran/dbt_ad_reporting).
 - Generates a comprehensive data dictionary of your source and modeled Facebook Ads data through the [dbt docs site](https://fivetran.github.io/dbt_facebook_ads/).
 
 <!--section="facebook_ads_transformation_model"-->
-The following table provides a detailed list of all models materialized within this package by default. 
+The following table provides a detailed list of all models materialized within this package by default.
```
Comment (fivetran-joemarkiewicz) [inline README.md:25]:
```suggestion
The following table provides a detailed list of all tables materialized within this package by default.
```

File: README.md [hunk 3]
```diff
@@ -14,15 +14,15 @@
...
 > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_facebook_ads/#!/overview?g_v=1&g_e=seeds).
 
 | **Model**                | **Description**                                                                                                                                |
```
Comment (fivetran-joemarkiewicz) [inline README.md:28]:
```suggestion
| **Table**                | **Description**                                                                                                                                |
```

**Review by fivetran-avinash — APPROVED** [review]
Removed an exclamation point and changed model to table, but everything else looked good!

**Pull request merged**